standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -857,6 +980,17 @@ public void commitTxn(CommitTxnRequest rqst)
               rs.getString(1), rs.getString(2), txnid,
               rs.getTimestamp(3, Calendar.getInstance(TimeZone.getTimeZone("UTC"))).getTime());
         }
+
+        if (rqst.isSetReplPolicy()) {
+          s = "delete from REPL_TXN_MAP where RTM_SRC_TXN_ID = " + sourceTxnId +
+                  " and RTM_REPL_POLICY = " + quoteString(rqst.getReplPolicy());
+          LOG.debug("Repl going to execute  <" + s + ">");
+          stmt.executeUpdate(s);
```

Comment:
Add an info when add/remove entries to REPL_TXN_MAP as customer will enable only that.

Reply:
changed the log to info